interface/main/messages/messages.php

```diff
@@ -350,16 +350,8 @@
                                 $noteid = $_GET['noteid'];
                             }
                             // Check to make sure the noteid is assigned to the user
```

Comment:
lines 349-351 are throwing me off a bit. it seems like below code (and original code) would break if $_GET['noteid'] is not set, yet that block above lets it run if not set. I'd prob instead do something like:
```
$noteid = (int)$_GET['noteid'];
if (empty($noteid)) {
    die("There was an error processing your request.");
}
```